spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -121,13 +121,25 @@
         'DocIssueType' => "Driver's License - STAR",
         'OrientationChanged' => 'true',
         'PresentationChanged' => 'false',
+        classification_info: {
+          Front: {
+            ClassName: 'Drivers License',
+          },
+          Back: {
+            ClassName: 'Drivers License',
+          },
+        },
       )
     end
 
     it 'notes that address line 2 was present' do
       expect(response.pii_from_doc).to include(address2: 'APT 3E')
       expect(response.to_h).to include(address_line2_present: true)
     end
+
+    it 'mark doc type as supported' do
```

Comment:
Is there a plan to add a test for when a non-supported doc type is submitted (or maybe it exists already)?

Reply:
@night-jellyfish, the same logic tested somewhere else, but adding the test anyway.